### ISSUE TITLE
Suggested values for the type attribute of note

### ISSUE THREAD

**Issue by katjameden** [clarin-eric/ParlaMint#384]
While checking the guidelines for valid type attribute values, I noticed that the [formal specifications](https://clarin-eric.github.io/ParlaMint/#TEI.note) for the element `note` only specify `type` attribute sample values (such as narrative and summary). 

The values, specific to ParlaMint (speaker, time, vote…) are only shown in the Examples. 

These values should be added (or maybe even replaced) to the suggested values specification.

**Comment by nuriabel** [comment]
Hi, we got and error for a note type="speaker", is the problem mentioned in this issue the reason?

ERROR
/mnt/d/UPF/proyecto_parlamint/ParlaMint/Data/ParlaMint-ES-CT/ParlaMint-ES-CT_2018-01-17-0101.xml:103:306: error: text not allowed here; expected element "gap", "incident", "kinesic", "note", "pb", "s" or "vocal"

XML:
            <note>ORDRE DEL DIA DE LA CONVOCATÒRIA</note>
            <note>Punt únic: Constitució del Ple del Parlament i elecció de la Mesa del Parlament (tram. 396-00001/12 i 398-00001/12).</note>
            <note type="speaker">El secretari general (Xavier Muro i Bas)</note>
            <u xml:id="ParlaMint-ES-CT_2018-01-17-0101.1" who="MuroXavier" ana="member" xml:lang="ca">

**Comment by TomazErjavec** [comment]
> Hi, we got and error for a , is the problem mentioned in this issue the reason?

No, this issue refers to the confusion about the `note/@type` values, but `note/@type` is not obligatory anyway, and the values are just recommended, not enforced through the schema.

You seem to be using plain text outside `<seg>`, at least as far as I can make out from the error message - it would help if you posted the offending snippet of your XML and best just open a separate issue for it.
